share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
```

Comment:
I think maybe "Give this address to those that you want to send you files" is better? They don't send you files yet until after you give them the address, and there's no guarantee that anyone will send you files even if you give them the address.

Reply:
I thought about: "Give out this address to people whom will send you files." but I dont think the distinction is clear enough. Was also for some reason trying to avoid "people", in case it is automated.
"Give this address to people wishing to send you files." is nice and short.
---
However "files", is an arbitration, and I would understand if this is an issue you are not sympathetic too, but in my head, "Give this address to people wishing to send you an OnionShare" is better, because you prepare an OnionShare, and you send it, done. You send a packet, not content. In this case "files", may be just the one file (?), which OnionShare covers. My assumption is that it is single.